acceptance_tests/features/view_collection_exercise.feature

```diff
@@ -0,0 +1,43 @@
+Feature: View Collection Exercise
```

Comment:
I think we could change the names of these feature files. 
This test is actually for a view of a single survey and the previous one was for viewing the list of surveys
The next story is around a single view of a collection exercise